lib/src/windows/file_picker_windows_ffi_types.dart

```diff
@@ -71,7 +71,7 @@ typedef GetSaveFileNameWDart = int Function(
 ///
 /// Reference:
 /// https://docs.microsoft.com/en-us/windows/win32/api/shlobj_core/ns-shlobj_core-browseinfoa
-class BROWSEINFOA extends Struct {
+final class BROWSEINFOA extends Struct {
```

Comment:
`dart:ffi` Structs are now supposed to be `base`, `sealed` or `final`.

Context: https://github.com/dart-lang/sdk/issues/51683